src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint.py

```diff
@@ -600,8 +604,7 @@ def check_inputs(
                     f" {negative_prompt_embeds.shape}."
                 )
 
-    # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline.prepare_latents
-    def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype, device, generator, latents=None):
+    def prepare_latents(self, image, timestep, batch_size, num_channels_latents, height, width, dtype, device, generator, is_strength_max, latents=None):
```

Comment:
We need to make this function backwards compatible:
```suggestion
    def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype, device, generator, image=None, timestep=None, is_strength_max=True, latents=None):
```

Reply:
Added your suggestion + a new additional check:

```
if (image is None or timestep is None) and not is_strength_max:
    raise ValueError(
        f"Since strength < 1. initial latents are to be initialised as a combination of Image + Noise."
        f"However, either the image or the noise timestep has not been provided."
    )
```

This ensures the method won't throw an unexpected error if image/timestep is not passed when strength < 1.